lib/Common/Core/ConfigParser.cpp

```diff
@@ -337,19 +337,70 @@ void ConfigParser::ParseConfig(HANDLE hmod, CmdLineArgsParser &parser)
         {
             return;
         }
+
         if (_wfopen_s(&configFile, filename, _u("r, ccs=UNICODE")) != 0 || configFile == nullptr)
         {
             return;
         }
     }
+#else
+    // Two-pathed for a couple reasons
+    // 1. PAL doesn't like the ccs option passed in.
```

Comment:
Sorry, wasn't clear- if you pass a mode string with "D", PAL prints out an error and returns NULL causing wfopen_s to fail. I'm going to leave the code as is for now, but I'll comment on the issue to point out the second issue - I agree that whoever fixes #3176 should fix both issues